cloudconfig/instancecfg/instancecfg.go

```diff
@@ -520,12 +520,12 @@ func (cfg *InstanceConfig) ToolsList() coretools.List {
 // all usage-sites are updated to pass through non-empty URLs.
 func (cfg *InstanceConfig) SetTools(toolsList coretools.List) error {
 	if len(toolsList) == 0 {
-		return errors.New("need at least 1 tools")
+		return errors.New("need at least 1 agent")
 	}
 	var tools *coretools.Tools
 	for _, listed := range toolsList {
 		if listed == nil {
-			return errors.New("nil entry in tools list")
+			return errors.New("nil entry in agents list")
```

Comment:
note that "agent binaries list" doesn't work very well either. list of agent binaries ?